src/map.cpp

```diff
@@ -3137,35 +3137,44 @@ void map::process_items( bool active, T processor, std::string signal )
 }
 
 template<typename T>
-void map::process_items_in_submap( submap *const current_submap, int gridx, int gridy, T processor,
-                                   std::string signal )
+void map::process_items_in_submap( submap *const current_submap, int const gridx, int const gridy,
+    T processor, std::string const &signal )
```

Comment:
I had to re-wrap the line after adding const; you'd rather the wrapped params line up with the start of that param list then?